src/config.py

```diff
@@ -17,6 +17,9 @@ def load_config(configfile):
     config['mail_alert_address'] = parser.get('settings', 'mail_alert_address')
     config['email_alert'] = parser.get('settings', 'email_alert')
     config['uptime_threshold'] = parser.get('settings', 'uptime_threshold')
-    config['DO_TOKEN'] = parser.get('settings', 'DO_TOKEN')
+    try:
```

Comment:
Not important, but you *could* also write this as
`config['DO_TOKEN'] = os.environ.get('DO_TOKEN', None) or parser.get('settings', 'DO_TOKEN')`

Or for extra credit:

```
settings = [ 'uptime_threshold', 'DO_TOKEN' ]   # And others
for setting in settings:
   config[setting] = os.environ.get(setting, None) or parser.get('settings', setting)
```